app/src/main/java/org/sopt/and/presentation/ui/signin/SignInViewModel.kt

```diff
@@ -42,13 +43,17 @@ class SignInViewModel : ViewModel() {
 
     fun loginUser() {
         viewModelScope.launch {
-            try {
-                val response = loginService.postLogin(RequestLoginDto(userId.value, userPassWord.value))
+            val result = loginUseCase.invoke(userId.value, userPassWord.value)
```

Comment:
```suggestion
            val result = loginUseCase(userId.value, userPassWord.value)
```
이렇게도 사용 가능해보이네요!! 추가로 invoke 함수에 대해 공부해보는 것도 좋을 것 같아요!!